README.md

```diff
@@ -118,6 +118,16 @@ interface TsserverOptions {
      * @default 'off'
      */
     trace?: 'off' | 'messages' | 'verbose';
+    /**
+     * Whether a dedicated server is launched to more quickly handle syntax related operations, such as computing diagnostics or code folding.
+     *
+     * Allowed values:
+     *  - auto: Spawn both a full server and a lighter weight server dedicated to syntax operations. The syntax server is used to speed up syntax operations and provide IntelliSense while projects are loading.
+     *  - never: Don't use a dedicated syntax server. Use a single server to handle all IntelliSense operations.
+     *
+     * @default 'auto'
+     */
+    useSyntaxServer?: 'auto' | 'never';
```

Comment:
It looks like it should be a boolean value but it's this way because VSCode also supports `always` and I want to have this future-proof in case support for `always` is added later.

To support `always` we'd need to add a bunch of checks to avoid triggering certain requests when it's used. Also, I believe in VSCode `always` is forced by the editor when opening single files and is not really meant to be used manually. So it might required some client glue to be usable in other editors.